Gemfile.lock

```diff
@@ -513,6 +515,7 @@ GEM
     zeitwerk (2.6.13)
 
 PLATFORMS
+  x86_64-darwin-21
   x86_64-linux
```

Comment:
This doesn't look right ... In normal circumstances, platform should be set to :ruby

Reply:
Agreed - not sure how that crept in. Fixed now.